README.md

```diff
@@ -479,6 +479,7 @@ Install HLS along with the following emacs packages:
 [lsp-mode](https://github.com/emacs-lsp/lsp-mode)
 [lsp-ui](https://github.com/emacs-lsp/lsp-ui)
 [lsp-haskell](https://github.com/emacs-lsp/lsp-haskell)
+use-package
```

Comment:
Even though I agree that providing info about use-package would be awesome, I think use-package lives in a different level from others' and should not be placed in the same list. One can use HLS without use-package (and only with lsp-mode/lsp-ui/lsp-haskell), but it will be greatly difficult to use HLS without lsp-mode, for example.

BTW, wherever we add this, I think we should also put the link to [the repo of use-package](https://github.com/jwiegley/use-package).

Reply:
I have removed use-package from that list and added the line

> Install [use-package](https://github.com/jwiegley/use-package), and add the following to your .emacs